src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -283,6 +283,11 @@ class ConnectionType(str, Enum):
     CUSTOM = "Custom"
 
 
+ALL_CONNECTION_TYPES = set(
```

Comment:
not sure if this is a good way to get names of all connection classes + @brynn-code

Reply:
It's ok, we made that deal everywhere XD.